packages/default-theme/cms/elements/CmsElementCategoryNavigation.vue

```diff
@@ -11,6 +11,29 @@
           :key="accordion.id"
           :header="accordion.name"
         >
+          <template #header="{header, isOpen, accordionClick, showChevron}">
+            <div class="cms-element-category-navigation__menu-item">
+              <nuxt-link
+                :to="getCategoryUrl(accordion.route)"
+                
+              >
+                <SfButton
+                  :aria-pressed="isOpen.toString()"
+                  :aria-expanded="isOpen.toString()"
+                  :class="{ 'sf-accordion-item__header--open': isOpen }"
+                  class="sf-button--pure sf-accordion-item__header"
+                >
+                {{ header }}
+                </SfButton>
+              </nuxt-link>
+              <SfChevron
+                tabindex="0"
+                class="sf-accordion-item__chevron"
+                :class="{ 'sf-chevron--right': !isOpen }"
+                @click.native="accordionClick"
```

Comment:
@patzick wdyt about this solution? I know it has drawbacks, so mayby you have better solution